### PR TITLE
update comment invitation readers to PC group

### PR DESCRIPTION
**Pull request by purujitgoyal**

### CODE REVIEW

**Review by melisabok — APPROVED** [review]
Awesome! @celestemartinez do you want to take a look?

File: openreview/venue_request/process/deployProcess.py [hunk 1]
```diff
@@ -390,4 +390,18 @@ def process(client, note, invitation):
             signatures = ['~Super_User1']
         )
 
-        client.post_invitation(matching_invitation)
\ No newline at end of file
+        client.post_invitation(matching_invitation)
```
Comment (celestemartinez) [inline openreview/venue_request/process/deployProcess.py:393]: Looks like the comment invitation's reply.readers are updated only when we setup the paper matching, it should be outside of the if.

Comment (purujitgoyal) [reply openreview/venue_request/process/deployProcess.py:393]: Fixed.

**Review by celestemartinez — APPROVED** [review]

**Comment by celestemartinez** [comment]
Looks good!

File: openreview/venue_request/process/deployProcess.py [hunk 2]
```diff
@@ -390,4 +390,18 @@ def process(client, note, invitation):
...
+
+    client.post_invitation(openreview.Invitation(
+        id=SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Comment',
+        super=SUPPORT_GROUP + '/-/Comment',
+        reply={
+            'forum': forum.id,
+            'referent': forum.id,
```
Comment (melisabok) [inline openreview/venue_request/process/deployProcess.py:400]: here is the issue, @purujitgoyal this should be `replyto` instead. 

Could you fix it and add a test?

Comment (melisabok) [reply openreview/venue_request/process/deployProcess.py:400]: `replyto` must be None in order the comment to be present in all the replies.

Comment (purujitgoyal) [reply openreview/venue_request/process/deployProcess.py:400]: what about `forum`?

Comment (purujitgoyal) [reply openreview/venue_request/process/deployProcess.py:400]: and test for adding a new PC and posting a comment?

Comment (melisabok) [reply openreview/venue_request/process/deployProcess.py:400]: forum should stay, the invitation should only replies to `forum.id` only.

> and test for adding a new PC and posting a comment?

That would be great, I added a comment in the new PR, you can also tests the existing replies were correctly updated.

**Pull request merged**